mautic/mautic
# Fix long tag name

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Long tag name is display without wrap in contact profile

Before

![image](https://user-images.githubusercontent.com/462477/102514659-06961e80-408d-11eb-831a-9f7bd771662f.png)

After

![image](https://user-images.githubusercontent.com/462477/102514753-24638380-408d-11eb-865b-c1bbe41740a2.png)





#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contact with long tag name 
3. Go to contact profile and see that tag is in one line without break to more lines
4. This PR fixed it
